app/views/courses/manage.html.erb

```diff
@@ -62,6 +62,9 @@
               <%= link_to "Export course", export_course_path(@course),
                           { title: "Export course configurations and assessments" } %>
             </li>
+            <li>
+              <%= link_to "File Manager", "#{file_manager_index_path}/#{@course.name}", { title: "File manager" } %>
```

Comment:
The path could be written as `path_file_manager_file_manager_index_path(@course.name)`, although it's a bit of a mouthful so maybe see if there's a way to shorten the name...

Reply:
Shortened with other file names